objc/objc_runtime_darwin_test.go

```diff
@@ -77,8 +77,7 @@ func ExampleID_SendSuper() {
 
 func TestSend(t *testing.T) {
 	// NSNumber comes from Foundation so make sure we have linked to that framework.
-	_, err := purego.Dlopen("Foundation.framework/Foundation", purego.RTLD_GLOBAL)
-	if err != nil {
+	if _, err := purego.Dlopen("Foundation.framework/Foundation", purego.RTLD_GLOBAL); err != nil {
```

Comment:
Should be `purego.RTLD_GLOBAL|purego.RTLD_LAZY` as NOW or LAZY is supposed to be used with GLOBAL/LOCAL